docs/layouts/shortcodes/guides/signup.md

```diff
@@ -0,0 +1,97 @@
+Depending on whether or not you have an existing project you want to migrate to Platform.sh,
+there are three ways to get started.
+
+### With a template
+
+Deploy a {{ .Get "name" }} template itself straight to Platform.sh.
+Use the button below to launch a {{ .Get "name" }} template project
+and create a Platform.sh account along the way.
+If you don't want to sign up with an email address,
+you can sign up using an existing GitHub, Bitbucket, or Google account.
+If you choose one of these options, you can set a password for your Platform.sh account later.
+
+<p align="center">
+  <a href='https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/{{ .Get "template" }}/.platform.template.yaml&utm_content={{ .Get "template" }}&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform'>
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+  </a>
+</p>
+
+Once the template is deployed, you can follow the rest of this guide
+so you can better understand the extra files and tweaks in the repository.
+
+### With a brand new code base
+
+You can start with a vanilla code base or push a pre-existing project to Platform.sh.
+
+1. [Register for a trial Platform.sh account](https://auth.api.platform.sh/register).
+
+   If you don't want to sign up with an email address, you can sign up using an existing GitHub, Bitbucket, or Google account
+   If you choose one of these options, you can set a password for your Platform.sh account later.
+
+1. Create your first project.
+
+   Since you're providing your own code, use the **Blank project** option.
```

Comment:
This block is reused at line 66, maybe worth including it once on the two spots?

Reply:
Okey since this page/block is included through `{{% /guides/signup %}}`, I assume my previous comment is irrelevant.

Reply:
I definitely thought about it, but decided I'd already spent too long on what was supposed to be a quick fix. Also, because it's already a shortcode and including a shortcode inside a shortcode isn't clear to me. (Most responses assume nesting, which isn't what I want.)